WooCommerce/Classes/ViewRelated/Orders/Order Details/Review Order/ReviewOrderViewController.swift

```diff
@@ -284,6 +343,100 @@ private extension ReviewOrderViewController {
         let billingInformationViewController = BillingInformationViewController(order: viewModel.order)
         navigationController?.pushViewController(billingInformationViewController, animated: true)
     }
+
+    /// Handle add tracking
+    ///
+    func addTrackingTapped() {
+        let addTrackingViewModel = AddTrackingViewModel(order: viewModel.order)
+        let addTracking = ManualTrackingViewController(viewModel: addTrackingViewModel)
+        let navController = WooNavigationController(rootViewController: addTracking)
+        present(navController, animated: true, completion: nil)
+    }
+
+    /// Handle shipment tracking tapped
+    ///
+    func shipmentTrackingTapped(at indexPath: IndexPath) {
+        guard let cell = tableView.cellForRow(at: indexPath) as? OrderTrackingTableViewCell else {
+            return
+        }
+
+        guard let tracking = viewModel.orderTracking(at: indexPath.row) else {
+            return
+        }
+
+        let actionSheet = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+        actionSheet.view.tintColor = .text
+
+        actionSheet.addCancelActionWithTitle(TrackingAction.dismiss)
+
+        actionSheet.addDefaultActionWithTitle(TrackingAction.copyTrackingNumber) { [weak self] _ in
+            self?.sendToPasteboard(tracking.trackingNumber, includeTrailingNewline: false)
+        }
+
+        if tracking.trackingURL?.isEmpty == false {
+            actionSheet.addDefaultActionWithTitle(TrackingAction.trackShipment) { [weak self] _ in
+                self?.openTrackingDetails(tracking)
```

Comment:
Q: Do we not track any analytics events here?

Reply:
I have taken a look at the Analytics sheet and saw that the analytics events for order fulfillment were deprecated since 5.6. I will post a question to ask the team about updated events for this new screen, but for now I guess we will not have analytics (I found [it's the same](https://github.com/woocommerce/woocommerce-android/pull/4086) on Android too).